Makefile

```diff
@@ -195,6 +195,16 @@ semgrep:
 	@semgrep --exclude "securedrop/tests/" --error --strict --metrics off --max-chars-per-line 200 --verbose --config "p/r2c-security-audit" securedrop
 	@echo
 
+
+# check dependencies in Cargo.lock
+.PHONY: rust-audit
+rust-audit:
+	@echo "███ Running Rust dependency checks..."
+	@cargo install cargo-audit
+	@cargo audit
```

Comment:
I typically run this with `cargo audit -D warnings` so e.g. yanked crates also cause a failure. But now that I re-read the documentation, we can create a `audit.toml` file (see https://github.com/rustsec/rustsec/blob/main/cargo-audit/audit.toml.example) to have yanked and unmaintained crates cause failures, plus we can limit advisories to Linux x86_64 (and not get stuff about Windows, etc.).

Reply:
added, and bumped `hermit-abi` as a result.